scripts/onboarding/templates/arc-k8s-extension-msi-auth/existingClusterOnboarding.json

```diff
@@ -20,12 +20,6 @@
         "description": "Azure Monitor Log Analytics Resource ID"
       }
     },
-    "workspaceRegion": {

```

Comment:
Please note that,  MSI auth support exists in k8s-extension of the azure-cli extensions so which needs to be updated as well.

Reply:
Will sync on this, with you.

Reply:
created draft PR with the change here -> https://github.com/AzureArcForKubernetes/azure-cli-extensions/pull/175